gitea/release.go

```diff
@@ -48,6 +49,33 @@ func (c *Client) GetRelease(user, repo string, id int64) (*Release, error) {
 	return r, err
 }
 
+// ListReleaseAttachments gets all the assets of a release in a repository
+func (c *Client) ListReleaseAttachments(user, repo string, id int64) ([]*Attachment, error) {
```

Comment:
These functions were removed in #77, because the API endpoints they use do not exist. I think we should only bring them back if we have plans to implement the corresponding API endpoints in the near future.

Reply:
Looks like we do have plans: https://github.com/go-gitea/gitea/pull/3075